MobileSyncExplorerKotlin/app/AndroidManifest.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.salesforce.samples.mobilesynccompose">
+
+    <application
+        android:name=".app.MobileSyncComposeApp"
```

Comment:
I don't think we need `.app`.

Reply:
He does because the MobileSyncComposeApp class is in a app subpackage.

Reply:
I guess what I really meant is do we need the `app` subpackage if it only contains our simple Application class?  I thought apps typically put this file in the root and not inside a subpackage.  Not a big deal either way.

Reply:
At one point I had more things pertaining to the Application class in that package, but they have since been removed.  I could do away with the `app` package if that would be preferred.